### PR TITLE
Newsletter 13: Add "Chargrid Roguelike Tutorial 2020"

### PR DESCRIPTION
**Pull request by stevebob**
https://github.com/rust-gamedev/rust-gamedev.github.io/issues/232

### CODE REVIEW

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -64,6 +64,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Learning Material Updates
 
+### [Chargrid Roguelike Tutorial 2020][chargrid-roguelike-tutorial-2020]
+
+![Chargrid Roguelike Tutorial 2020](chargrid-roguelike-tutorial-2020.png)
+
+[Chargrid][chargrid] by [@stevebob][] is a collection of crates for building
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:71]:
```suggestion
[Chargrid][chargrid] by [@stevebob] is a collection of crates for building
```
This isn't needed.

**Review by 17cupsofcoffee — APPROVED** [review]
Tiny formatting nitpick, but looks good otherwise.

**Review by ozkriff — APPROVED** [review]
(made a small style tweak)

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**